tests/ui/traits/const-traits/call-generic-method-fail.stderr

```diff
@@ -3,6 +3,11 @@ error[E0277]: the trait bound `T: [const] PartialEq` is not satisfied
    |
 LL |     *t == *t
    |     ^^^^^^^^
+   |
+note: trait `PartialEq` is implemented but not `const`
+  --> $SRC_DIR/core/src/ptr/const_ptr.rs:LL:COL
+note: trait `PartialEq` is implemented but not `const`
+  --> $SRC_DIR/core/src/ptr/mut_ptr.rs:LL:COL
```

Comment:
Why is it pointing at the raw ptr impls?

Reply:
I think that they are incorrectly detected as `CandidateSimilarity::Exact`, which is incorrect. I can skip them explicitly on this PR by adding some other check, or look into it at another time.